macros/FirefoxOSSidebar.ejs

```diff
@@ -13,115 +13,114 @@ function currentPageIsUnder(root) {
 }
 
 var text = mdn.localString({
-  'en-US': {
-    'Build_and_install': 'Build and install',
-    'Build_and_install_overview': 'Build and install overview',
-    'B2G_OS_build_process_summary': 'B2G OS build process summary',
-    'Build_prerequisites': 'Build prerequisites',
-    'Preparing_for_your_first_build': 'Preparing for your first build',
-    'Building_B2G_OS': 'Building B2G OS',
-    'B2G_installer_add-on': 'B2G installer add-on',
-    'Building_B2G_OS_for_Flame_on_Mac_OS_X': 'Building B2G OS for Flame on Mac OS X',
-    'Choosing_how_to_run_Gaia_or_B2G_OS': 'Choosing how to run Gaia or B2G OS',
-    'Compatible_Devices': 'Compatible Devices',
-    'Installing_B2G_OS_on_a_mobile_device': 'Installing B2G OS on a mobile device',
-    'Creating_and_applying_B2G_OS_update_packages': 'Creating and applying B2G OS update packages',
-    'Building_and_installing_FOTA_community_builds': 'Building and installing FOTA community builds',
-    'B2G_build_variables_reference_sheet': 'B2G build variables reference sheet',
-    'Porting_B2G_OS': 'Porting B2G OS',
-    'Porting_overview': 'Porting overview',
-    'Porting_basics': 'Porting basics',
-    'Porting_on_CyanogenMod': 'Porting on CyanogenMod',
-    'Developing_Gaia': 'Developing Gaia',
-    'Developing_Gaia_overview': 'Developing Gaia overview',
-    'Running_the_Gaia_codebase': 'Running the Gaia codebase',
-    'Understanding_the_Gaia_codebase': 'Understanding the Gaia codebase',
-    'Making_Gaia_code_changes': 'Making Gaia code changes',
-    'Testing_Gaia_code_changes': 'Testing Gaia code changes',
-    'Submitting_a_Gaia_patch': 'Submitting a Gaia patch',
-    'Gaia_build_system_primer': 'Gaia build system primer',
-    'Make_options_reference': 'Make options reference',
-    'Gaia_tools_reference': 'Gaia tools reference',
-    'B2G_OS_APIs': 'B2G OS APIs',
-    'Mulet' : 'Run Gaia on desktop using Mulet', 
-    'Different_ways_to_run_Gaia' : 'Different ways to run Gaia' 
+{
```

Comment:
This opening curly brace and the corresponding closing one are redundant and need to be removed.

Sebastian